README.md

```diff
@@ -502,7 +502,8 @@ is created, and thereafter will increase as new shares are added (issued) and gr
 people in exchange for money or services.
 Outstanding shares may increase or decrease for other reasons too (such as stock splits
 and share buybacks, which we won’t get into here).
-Later, we discuss [several subtleties](#counting-shares) in how shares are counted.
+
+- Later, we discuss [several subtleties](#counting-shares) in how shares are counted.
```

Comment:
Remove bullet? Seems like it's a regular paragraph/sentence.

Reply:
Fixed.